### ISSUE TITLE
Creating job Failed. Cant seem to get reverse shell back.

### ISSUE THREAD

**Issue by CyberJunnkie** [nathan31337/Splunk-RCE-poc#2]
So i have tried installing splunk 9.0.1 and also 9.1.1 as stated in previous issue by you and also on your blog. I have tried both ways (Installing it on ubuntu and via docker too).  I am getting same error. I am attaching a screenshot below. The upper one is on on prem version and below one was tried on docker as stated bu you in issue 1

![splunkissue](https://github.com/nathan31337/Splunk-RCE-poc/assets/76212264/149a3e7a-72f3-4ef7-9fd3-fc95d88ffca0)

**Comment by malladisiddu** [comment]
I have faced similar issue, ensure that you have atleast 5 gb of free space. I got it fixed that way.

**Comment by nathan31337** [comment]
Yep, I also ran into that issue with low disk space. I recommend proxying the request through burpsuite to debug.
